docs/core/porting/windows-compat-pack.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Porting to .NET Core - Using the Windows Compatibility Pack
+description: Porting to .NET Core - Using the Windows Compatibility Pack
+keywords: .NET, .NET Core, Windows, Compatibility
+author: immol
+ms.author: mairaw
+ms.date: 11/13/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: ffa28aed-45dd-4b02-965f-768c467829c3
+---
+
+# Using the Windows Compatibility Pack
+
+Many customers have reported that the primary reason why they have trouble
+porting to .NET Core is that they depend on APIs and technologies that only
+exist in .NET Framework. The *Windows Compatibility Pack* is about providing a
+good chunk of these technologies so that building .NET Core applications as well
+as .NET Standard libraries becomes much more viable for existing code.
+
+This is a logical [extension of .NET Standard 2.0][API Expansion] which
+significantly increased the API set to ensure code largely already compiles
+as-is. But in order to keep .NET Standard understandable, this didn't include
+technologies that can't work across all platforms, such as registry, WMI, or
+reflection emit APIs.
+
+The *Windows Compatibility Pack* sits above .NET Standard and is thus free to
+provide access to technologies that are Windows only. It is especially useful
+for customers that want to move to .NET Core but plan to stay on Windows as a
+first step. In that scenario, not being able to use Windows-only technologies is
+only a migration hurdle with zero architectural benefits.
+
+## Contents
+
+The *Windows Compatibility Pack* is provided via the NuGet Package
+[Microsoft.Windows.Compatibility] and can be referenced from .NET Core as well
+as .NET Standard.
```

Comment:
should we add a noun in the end? can be referenced from .NET Core as well as .NET Standard projects/code?